main/modes/touchTest/touchTest.h

```diff
@@ -5,4 +5,91 @@
 
 extern swadgeMode_t touchTestMode;
 
+/**
+ * @brief Draw a circular representation of the touchpad
+ *
+ * @param x X coordinate on the TFT for the center of the circle
+ * @param y Y coordinate on the TFT for the center of the circle
+ * @param r Radius of the circle
+ * @param segs Number of directional segments to split the touchpad into
+ * @param center True to include a center zone, false to only use directional segments
+ * @param touched Whether the touchpad is currently being touched
+ * @param val The output you received from a prior call to `getTouchJoystickZones`
+ */
+static void touchDrawCircle(font_t* font, const char* label, int16_t x, int16_t y, int16_t r, int16_t segs, bool center,
```

Comment:
No no no no. Do not put functions in headers. Ever. Well, except for `inline` but that's special. Each time a header is included it's going to create a new copy of this function in source code. You made them `static` so they won't collide, but this is awful for code space.

Reply:
Oh, I thought that was the case but it HATED when I tried to declare the function in the header and define it in the c file, `extern` or not

Reply:
Ah, looks like `static` was my problem. Boy did Java set me up for failure when it comes to that keyword; I can never remember which purpose it serves in which language 😭

Reply:
Yeah, static is weird in C. A static function can only exist in it's own file. A static variable outside a function is the same. A static variable in a function persists it's value between function calls. Be careful